crux-llvm/test/Test.hs

```diff
@@ -408,13 +409,18 @@ mkTest sweet _ expct =
           Info.arch /= "x86_64" &&
           TS.rootBaseName sweet == "T972-fail"
 
+    -- T816 has different output on GHC 9.10.1, see #1377.
+    let skipGhc910T816 =
+          Info.compilerVersion == Version.makeVersion [9, 10] &&
+          TS.rootBaseName sweet == "T816"
```

Comment:
Please do file a separate issue for this, as this is a workaround that we will have to continuously update as we support newer GHC releases (unless we do something else to resolve it).

Reply:
https://github.com/GaloisInc/crucible/issues/1379